src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -513,14 +512,18 @@ public String toString()
 
             });
           }
-        }
-        catch (IOException ie) {
-          l.handleException(ie, DefaultEventLoop.this);
+        } catch (UnresolvedAddressException e) {
+          logger.error("Inet Address {} is not resolvable.", address, e);
```

Comment:
+1 for logging this specific exception within netlet at error level

It is very hard to diagnose/support if we need to ask the user to change configuration to see the error. It is infrequent and important to see, using error level seems appropriate. Maybe add a comment to that effect?